### DISCUSSION TITLE
mobile app open local bim file

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1506] [Q&A]
now, i download the bim file to my mobile devices(for exmple iphone ,android phone), and
i want open the bim file from mobile devices by imdeljs ,( mobile devices offline to server，because the mobile signal is not good usually )

**Comment by pmconne** [comment]
Mobile devices are not yet officially supported. Internally we have had support for iOS for a while and very recently added support for Android. Official support will come in the near future with sample applications, documentation, and UI libraries that work well on mobile form factors.

For the time being, if you want to try to get something running on mobile, you'll have to figure it out for yourself. [display-test-app](https://github.com/imodeljs/imodeljs/tree/master/test-apps/display-test-app/ios) can be built and run on an iPad, so that may be a good place to start.
But I recommend waiting for official support, samples, and documentation.
